### PR TITLE
[Documentation] The one with all the explaination

### PR DESCRIPTION
**Pull request by Sqyphen**
## Description

Based on feedback on previous PR's - this adds some more detailed documentation to the project

## Implementation

- Added more detail for setup from scratch
- Added notes about the process of handling and responding to requests
- Added further development notes

## Screenshot

n/a

## .env changes

n/a

## Additional requirements

n/a

Read my docs plz

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,145 @@
...
+        - <project_url>/submit
+    - Go to _Install App_
+      - Click **Install App to Workspace**
+      - Authorise it to work in FT workspace
+      - Copy the _Bot User OAuth Access Token_ and put it in the .env file for **SLACK_TOKEN**
+- Now, as long as your server is running, you will be able to receive Slack commends and access the Google Worksheet of your choosing/creation
+
+### Live
+
+Copy the same steps as Live but use Live URLs for each of the endpoints, interactive components etc
```
Comment (railsagainstignorance) [inline README.md]: same steps as Local

**Comment by railsagainstignorance** [comment]
LGTM. 
Once this is done, I'll have another look.
Could probably add some example outputs in the README (which I could do).

File: README.md (hunk 1 above)
Comment (Sqyphen) [reply README.md]: Ah yes, corrected now.
Thank you

**Comment by Sqyphen** [comment]
> LGTM.
> Once this is done, I'll have another look.
> Could probably add some example outputs in the README (which I could do).

**Comment by Sqyphen** [comment]
uhhh...didn't mean to close this...what?!

**Pull request merged**